cypress/integration/Authentication.feature

```diff
@@ -1,11 +0,0 @@
-Feature: Authentication
```

Comment:
Remove this entirely as this simple login test is now duplicated all over the place